### DISCUSSION TITLE
calculate FID and IS when test with pix2pixel

### DISCUSSION THREAD

**Discussion by wwjwy** [open-mmlab/mmediting#1417] [How-to]
Hi,
Can you show me an correct example of calculating the metrics of FID and IS when testing with pix2pix? I modified the config to "test_cfg = dict(direction='b2a', show_input=True,metrics=['FID', 'IS'])", but it does not work?
 Thanks!

**Comment by LeoXing1996** [comment]
Hey, @wwjwy , which branch are you currently using? Can you provide the complete config you used to evaluate FID and IS?

**Reply by wwjwy** [reply]
I used the master branch.
I only modified the config in line 29 to "test_cfg = dict(direction='b2a', show_input=True,metrics=['FID', 'IS'])".   https://github.com/open-mmlab/mmediting/blob/master/configs/synthesizers/pix2pix/pix2pix_vanilla_unet_bn_1x1_80k_facades.py
Then i run: python tools/test.py  config  latest.pth     but the value of FID and IS were not shown.
by the way, during the training, i did not modified the config. After training, i want to see the metrics, so i modified the config. Thanks!
